test/pseudo-tty/test-handle-wrap-isrefed-tty.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+// see also test/parallel/test-handle-wrap-isrefed.js
+
+const common = require('../common');
+const strictEqual = require('assert').strictEqual;
+
+function makeAssert(message) {
+  return function(actual, expected) {
+    strictEqual(actual, expected, message);
+  };
+}
+
+const assert = makeAssert('hasRef() not working on tty_wrap');
+
+const ReadStream = require('tty').ReadStream;
+const tty = new ReadStream(0);
+const isTTY = process.binding('tty_wrap').isTTY;
+assert(isTTY(0), true);
+assert(Object.getPrototypeOf(tty._handle).hasOwnProperty('hasRef'), true);
+assert(tty._handle.hasRef(), true);
+tty.unref();
+assert(tty._handle.hasRef(), false);
+tty.ref();
+assert(tty._handle.hasRef(), true);
```

Comment:
I feel like it might be better to leave it as is for consistency with parallel/test-handle-wrap-isrefed.js. This test case is basically one part of that file that needed to be separated out as it needs a TTY.